source/Grid/Grid.js

```diff
@@ -63,6 +64,12 @@ export default class Grid extends Component {
     noContentRenderer: PropTypes.func.isRequired,
 
     /**
+     * Optional callback invoked on mouse-wheel activity.
+     * (event: SyntheticEvent): void
+     */
+    onWheel: PropTypes.func,
```

Comment:
I'd like to remove this change for the time being. Markus' response on Twitter made me more anxious. This seems like the sort of change that, if we do, we should try in total isolation. (I'd also want to test it a lot more before I felt comfortable.)